tests/api_resources/abstract/test_deletable_api_resource.py

```diff
@@ -5,66 +5,73 @@ class TestDeletableAPIResource(object):
     class MyDeletable(stripe.api_resources.abstract.DeletableAPIResource):
         OBJECT_NAME = "mydeletable"
 
-    def test_delete_class(self, request_mock):
-        request_mock.stub_request(
+    def test_delete_class(self, http_client_mock):
+        http_client_mock.stub_request(
             "delete",
-            "/v1/mydeletables/mid",
-            {"id": "mid", "deleted": True},
+            path="/v1/mydeletables/mid",
+            rbody='{"id": "mid", "deleted": true}',
             rheaders={"request-id": "req_id"},
         )
 
         obj = self.MyDeletable.delete("mid")
 
-        request_mock.assert_requested("delete", "/v1/mydeletables/mid", {})
+        http_client_mock.assert_requested(
+            "delete", path="/v1/mydeletables/mid", query_string=""
+        )
         assert obj.deleted is True
         assert obj.id == "mid"
 
         assert obj.last_response is not None
         assert obj.last_response.request_id == "req_id"
 
-    def test_delete_class_with_object(self, request_mock):
-        request_mock.stub_request(
+    def test_delete_class_with_object(self, http_client_mock):
+        http_client_mock.stub_request(
             "delete",
-            "/v1/mydeletables/mid",
-            {"id": "mid", "deleted": True},
+            path="/v1/mydeletables/mid",
+            rbody='{"id": "mid", "deleted": true}',
             rheaders={"request-id": "req_id"},
         )
 
         obj = self.MyDeletable.construct_from({"id": "mid"}, "mykey")
 
         self.MyDeletable.delete(obj)
 
-        request_mock.assert_requested("delete", "/v1/mydeletables/mid", {})
+        http_client_mock.assert_requested(
+            "delete", path="/v1/mydeletables/mid", query_string=""
```

Comment:
is query_string a required parameter?

Reply:
No, it isn't. `http_client_mock .assert_requested()` will not check `query_string` if it's not passed in, so I specified it because I thought we might as well be stricter about the checks.